spyder/plugins/explorer/widgets/remote_explorer.py

```diff
@@ -126,6 +162,24 @@ def on_conf_update(self, option, value):
         elif option == "single_click_to_open":
             self.set_single_click_to_open(value)
 
+    def _on_show_context_menu(self, position):
+        index = self.view.indexAt(position)
+        source_index = self.proxy_model.mapToSource(self.view.currentIndex())
+        data_index = self.model.index(source_index.row(), 0)
+        data = self.model.data(data_index, Qt.UserRole + 1)
+
+        if not index.isValid():
+            return
+
+        if not data:
+            return
+
+        if data.get("type", "directory") == "directory":
+            return
```

Comment:
We need to give users a way to download entire directories. Otherwise, it'd be very inefficient to download many files in a single directory.

For that we'll need additional support in spyder-remote-services, as we talked about.

Reply:
Does that functionality need to be implemented in this PR or should I add here something like a TODO comment to implement that later in another PR?